src/components/Project/ImageAnalysis/ImageDownloadMenu.vue

```diff
@@ -0,0 +1,64 @@
+<script setup>
+import { onMounted } from 'vue'
+import { useAlertsStore } from '@/stores/alerts'
+
+const props = defineProps(['image', 'tifUrl'])
+
+const emit = defineEmits(['analysisAction'])
+
+const alertStore = useAlertsStore()
+
+function downloadLink(link, filename, fileType) {
+  if(!link){
+    alertStore.setAlert('error', `No ${fileType} available for download`)
+    return
+  }
+  const a = document.createElement('a')
+  a.href = link
+  a.download = filename
+  document.body.appendChild(a)
+  a.click()
+  document.body.removeChild(a)
+}
+
+onMounted(() => {
+  const getTifInput = {
+    'basename': props.image.basename
+  }
+  emit('analysisAction', 'get-tif', getTifInput)
+})
+
+</script>
+<template>
+  <v-speed-dial
+    location="left center"
+    transition="fade-transition"
+  >
+    <template #activator="{ props: activatorProps }">
+      <v-btn
+        v-bind="activatorProps"
+        icon="mdi-download"
+      />
+    </template>
+    <v-btn
+      key="1"
+      text=".FITS"
+      @click="downloadLink(props.image.url, props.image.basename, 'FITs')"
+    />
+    <v-btn
+      key="2"
+      text=".TIF"
+      @click="downloadLink(props.tifUrl, props.image.basename, 'TIF')"
+    />
+    <v-btn
+      key="3"
+      text=".JPG"
+      @click="downloadLink(props.image.largeCachedUrl, props.image.basename, 'JPG')"
```

Comment:
I think we might want to get the jpeg from the server as well, same way as the tiff... The largest jpeg we generate for the view is like 900px, but the full image might have a much higher resolution and I think downloading the jpeg you would want the full resolution. But again get it on demand rather than pre-generate it for every image.

Reply:
I think we're going to replace the carousel with the grid in which case theres not going to be any use for the non full sized images. can we replace the largeimageUrl that comes with the image to be a fullsized jpg? or is it better to save the space, in which case I'll go with requesting a full size in a new feature

Reply:
I would think we always want to have "large" but not full size images for the frontend use, since tablets don't have that high resolution and you don't need to see full rez on screen anyway... But if we did full rez everywhere it would be using a lot more bandwidth. You can run it past everyone else, but I think we should just have downloading the image in jpeg/tiff happen in the backend with a full size version, but keep the UI using the large not full size versions.